packages/nouns-assets/src/image-data.json

```diff
@@ -239,7 +239,9 @@
     "f8ce47",
     "395ed1",
     "ffc5f0",
-    "d4cfc0"
+    "d4cfc0",
+    "f9fafb",
+    "e5e7eb"
```

Comment:
where did those come from?

onchain data stops at `d4cfc0`. ref: https://assets.noundry.wtf/nouns/image-data.json

Reply:
Those are generated by `encode` script if I’m not mistaken, I've added those but didn't verify with onchain data.
https://github.com/nounsDAO/nouns-monorepo/blob/9a95303960098cae713ad58811c63f021250c89a/packages/nouns-assets/package.json#L11